### DISCUSSION TITLE
Cursor movement and selection in fuzzy finder

### DISCUSSION THREAD

**Discussion by waldauf** [helix-editor/helix#8284] [Ideas]
Hi folks!

I use [FZF](https://github.com/junegunn/fzf) for searching files in the command line or to filter objects from the list created by some command. FZF is set as ([documentation](https://github.com/junegunn/fzf#using-the-finder)):
- When you type keys or shift+keys, you're filtering.
- When you hold Ctrl/Control and `hjkl` keys you move with the cursor in the filtered field.
- With tab key you mark items in the filtered list.

Is it possible to set Helix in this way? Because in Helix you have to use arrows for moving in the filtered field which means to move your hand from keys to arrows. Also, the possibility of having a multi-selection for opening multiple files is a great feature.

Thank you for your opinion.

**Comment by kinoute** [comment]
I was wondering the same regarding multiple files selection in the fuzzy finder, it is not available?
